src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs

```diff
@@ -144,7 +145,17 @@ public async Task DisplayBannerAsync()
                     var version = (AssemblyInformationalVersionAttribute?)Assembly.GetExecutingAssembly().GetCustomAttribute(typeof(AssemblyInformationalVersionAttribute));
                     if (version is not null)
                     {
-                        stringBuilder.AppendLine(CultureInfo.InvariantCulture, $"Version: {version.InformationalVersion}");
+                        stringBuilder.Append(CultureInfo.InvariantCulture, $"Version: {version.InformationalVersion}");
+
+                        var buildTime = Assembly.GetExecutingAssembly().GetCustomAttributes(typeof(AssemblyMetadataAttribute))
+                        .OfType<AssemblyMetadataAttribute>().FirstOrDefault(x => x.Key == BUILDTIME_ATTRIBUTE_NAME);
+                        if (buildTime is not null)
+                        {
+                            DateTime buildDateTime = DateTime.FromOADate(double.Parse(buildTime.Value!, CultureInfo.InvariantCulture));
```

Comment:
Might be worth doing a null check and `TryParse`, isn't it?

Reply:
it cannot fail it's a double to string

Reply:
Try[ing] it